tests/op_config.json

```diff
@@ -222,7 +222,7 @@
   "id_token": {
     "class": "oidcop.id_token.IDToken",
     "kwargs": {
-      "default_claims": {
```

Comment:
I still see a disalignment on which variable name we must use.
I found this in op_config and base_claims in test.

I'm using base_claims, no problem to switch to default_claims, just need to decide which one to use and then a final alignment of code and documentation must be done

Reply:
Should always be base_claims. I think I went through the code base yesterday and fixed that everywhere.